raft.go

```diff
@@ -881,6 +881,8 @@ func (r *Raft) checkLeaderLease() time.Duration {
 				} else {
 					r.logger.Debug("failed to contact", "server-id", server.ID, "time", diff)
 				}
+
+				r.observe(FailedHeartbeatObservation{PeerID: server.ID, LastContact: f.LastContact()})
```

Comment:
Did you see my comment on the PR? I'm concerned that this only applies to voters (see line 865). When instead if this was done in the `heartbeat` method it would apply to any server.

Reply:
Yea, I'll add that in